### DISCUSSION TITLE
Components should have __private attributes that are not exposed to Js

### DISCUSSION THREAD

**Discussion by nerdoc** [django-commons/django-unicorn#681] [Ideas]
The current approach with [exclude](https://www.django-unicorn.com/docs/views/#exclude) is neat and does what it should. But it is not very intuitive.

What would be more interesting is adding support for _protected and/or __private attributes of a component. These should automatically **not** be exposed to the javascript frontend, but just usable in the backend.

**Comment by nerdoc** [comment]
AAARGH. This is already possible. Sorry again for the spam.

**Reply by adamghill** [reply]
No worries! Should I call out excluding private attributes more prominently in the docs?

**Reply by nerdoc** [reply]
Definitely!
